<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetViewGroupEditor</class>
 <widget class="QWidget" name="PresetViewGroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="10" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Group interval(ms)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="spinBoxItemInterval">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>Add(update) Group</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Colums(split with space)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="13">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonUp">
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDown">
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete Selected</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Untitled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEditColumns">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string notr="true">1 2 3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkBoxDisableGroup">
     <property name="text">
      <string>Disable the group</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QSpinBox" name="spinBoxGroupInterval">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Item interval(ms)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditColumns</tabstop>
  <tabstop>spinBoxItemInterval</tabstop>
  <tabstop>pushButtonAdd</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>spinBoxGroupInterval</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
